packages/sources/cfbenchmarks/src/endpoint/crypto-router.ts

```diff
@@ -6,10 +6,10 @@ import {
 } from '@chainlink/external-adapter-framework/util'
```

Comment:
This might be me just nitpicking but I don't think we need to add the `-router` to the end of the endpoint anymore. Think it made sense to when the endpoint and transports were in the same folder to help differentiate what's what. Since they're separated now we could just use the endpoint's name like `crypto`? What do you think?

Reply:
Initially i wanted to sort of highlight that this is a router endpoint, that's why there are multiple transports but now, especially after working with EA generator, i think it makes more sense to get rid of that suffix. I will update the PRs